service-deployment/bootstrap/frameworklauncher/frameworklauncher.yaml.template

```diff
@@ -62,7 +62,7 @@ spec:
       volumes:
       - name: launcher-log
         hostPath:
-          path: /mnt/sdb1/logs
+          path: {{ clusterinfo[ 'dataPath' ] }}/launcherlogs
```

Comment:
>auncherlogs [](start = 49, length = 11)

will the change from logs to launcherlogs affect frameworklauncher? please double check with @Yuqi

Reply:
The log folder in launcher's logic is container's file system. The host's path change won't affect it.